client/packages/invoices/src/InboundShipment/Dashboard/Widget.tsx

```diff
@@ -25,49 +31,72 @@ export const InboundShipmentWidget: React.FC<PropsWithChildrenOnly> = () => {
     }
   );
 
+  const { mutate: onCreate } = useInbound.document.insert();
+
   return (
-    <Widget title={t('inbound-shipments')}>
-      <Grid
-        container
-        justifyContent="flex-start"
-        flex={1}
-        flexDirection="column"
-      >
-        <Grid item>
-          {!hasError && (
-            <StatsPanel
-              isLoading={isLoading}
-              title={t('inbound-shipments')}
-              stats={[
-                {
-                  label: t('label.today', { ns: 'dashboard' }),
-                  value: formatNumber.round(data?.today),
-                },
-                {
-                  label: t('label.this-week', { ns: 'dashboard' }),
-                  value: formatNumber.round(data?.thisWeek),
-                },
-              ]}
-            />
-          )}
-        </Grid>
+    <>
+      {modalControl.isOn ? (
+        <InternalSupplierSearchModal
+          open={modalControl.isOn}
+          onClose={modalControl.toggleOff}
+          onChange={async name => {
+            modalControl.toggleOff();
+            try {
+              await onCreate({
```

Comment:
same concern as the stock widget - this won't be awaiting as expected. not an issue, as there's no code following, but a possible problem later.